network/src/counters.rs

```diff
@@ -121,7 +149,7 @@ pub static LIBRA_NETWORK_DIRECT_SEND_BYTES: Lazy<HistogramVec> = Lazy::new(|| {
 pub static PENDING_NETWORK_NOTIFICATIONS: Lazy<IntCounterVec> = Lazy::new(|| {
     register_int_counter_vec!(
         "libra_network_pending_network_notifications",
-        "Counters(queued,dequeued,dropped) related to pending inbound network notifications",
+        "State of pending inbound network notifications",
```

Comment:
I do wonder how meaningful this really is since we're talking about 15 seconds between metric pulls...

Reply:
Yeah and there's only 1 minute granularity through Grafana.

I think this is also not a valid description now that I look at it, it's not the state of pending inbound network notifications, but the number of network notifications that were processed at different states.